From: Li zeming <zeming@nfschina.com>
Subject: usr/gen_init_cpio.c: remove unnecessary -1 values from int file
Date: Mon, 19 Sep 2022 09:44:06 +0800

The file variable is assigned first, it does not need to be initialized.

Link: https://lkml.kernel.org/r/20220919014406.3242-1-zeming@nfschina.com
Signed-off-by: Li zeming <zeming@nfschina.com>
Cc: Li zeming <zeming@nfschina.com>
Cc: Masahiro Yamada <masahiroy@kernel.org>
Cc: Nicolas Schier <nicolas@fjasle.eu>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 usr/gen_init_cpio.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/usr/gen_init_cpio.c~usr-remove-unnecessary-1-values-from-int-file
+++ a/usr/gen_init_cpio.c
@@ -326,7 +326,7 @@ static int cpio_mkfile(const char *name,
 	char s[256];
 	struct stat buf;
 	unsigned long size;
-	int file = -1;
+	int file;
 	int retval;
 	int rc = -1;
 	int namesize;
_
